lib/msal-browser/test/encode/Base64Decode.spec.ts

```diff
@@ -95,6 +95,25 @@ describe("Base64Decode.ts Unit Tests", () => {
             );
         });
 
+        it("Decode idtokenclaims with special characters", () => {
```

Comment:
Do we already have corresponding encode test?

Reply:
Added a test for it in the encoding spec.